tvb_library/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN conda install python=3.7 nomkl numba scipy numpy networkx scikit-learn cytho
 
 RUN apt-get install -y zip unzip
 RUN cd /opt \
- && wget -q https://zenodo.org/record/10128131/files/tvb_data.zip?download=1 -O tvb_data.zip \
+ && wget -q https://zenodo.org/record/14992335/files/tvb_data.zip?download=1 -O tvb_data.zip \
  && mkdir tvb_data \
  && unzip tvb_data.zip -d tvb_data \
```

Comment:
line 16 here

Reply:
solved and an unrelated question: how come we still use python 3.7 here? shouldn't we upgrade?

Reply:
I believe this container is left from some old deployment, not used any more